src/utils/devtools.ts

```diff
@@ -32,58 +34,69 @@ export const devtools = <T extends object>(proxyObject: T, name?: string) => {
   let isTimeTraveling = false
   const devtools = extension.connect({ name })
   const unsub1 = subscribe(proxyObject, (ops) => {
-    const action = ops.map(([op, path]) => `${op}:${path.join('.')}`).join(', ')
+    const action = ops
+      .flatMap(([op, path]) =>
+        path.some((x) => typeof x === 'symbol')
+          ? []
+          : [`${op}:${path.join('.')}`]
+      )
+      .join(', ')
+
+    if (!action) {
+      return
+    }
     if (isTimeTraveling) {
       isTimeTraveling = false
     } else {
+      const proxyWithoutDevtools = Object.assign({}, proxyObject)
+      delete (proxyWithoutDevtools as any)[DEVTOOLS]
+
       devtools.send(
         `${action} - ${new Date().toLocaleString()}`,
-        snapshot(proxyObject)
+        proxyWithoutDevtools
```

Comment:
You can't skip `snapshot()`. Please try removing DEVTOOLS after taking a snapshot.